sentences/es/_common.yaml

```diff
@@ -51,6 +51,7 @@ expansion_rules:
   area: "[en | en el | en la | la | de la | del | de | de el | el] {area}"
   brightness: "{brightness} [porciento]"
   enciende: "(enciende | activa)"
+  pon: "(pon | poner | sube | subir | baja | bajar | ajusta | ajustar | configura | configurar)"
```

Comment:
Does it make sense to include the infinitives here? Personally I don't think anyone would talk like that.

Reply:
I would not talk like that, but i know some people who do. Each person has a digfferent style of talking to computers.

Reply:
Yes, for example my wife say "subir persianas de comedor".

Reply:
I stand corrected then 🤷‍♂️